src/components/EventForm.js

```diff
@@ -80,7 +84,13 @@ export default class EventForm extends React.Component {
       <form onSubmit={this.onSubmit}>
         <label htmlFor="title">
           Title
-          <input placeholder="Title" name="title" id="title" onChange={this.onTitleChange} />
+          <input
+            placeholder="Title"
+            name="title"
+            id="title"
```

Comment:
Check if you cannot use `id` in discussed handlers instead of creating the similar `name` props.